internal/server/handlers_certificate_requests.go

```diff
@@ -102,11 +80,9 @@ func CreateCertificateRequest(env *HandlerConfig) http.HandlerFunc {
 			writeError(w, http.StatusInternalServerError, "Internal Error")
 			return
 		}
-		certificateRequestResponse := CreateCertificateRequestResponse{
-			ID: int(id),
-		}
+		successResponse := SuccessResponse{Message: "success"}
 		w.WriteHeader(http.StatusCreated)
-		err = writeJSON(w, certificateRequestResponse)
+		err = writeJSON(w, successResponse)
```

Comment:
I'm not necessarily against all of those HTTP API changes but why include those in this PR, they are quite unrelated to the introduction of sqlair

Reply:
the db sqlair response no longer returned any integer for me to pass along to the http server. My other option was to return a fake 0 for ids for everything instead, which meant that we would have committed a version of notary where the responses for the API were meaningless.

Reply:
Ok yeah no we don't want to return a field that means nothing either. Let's keep it as is